src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -2011,7 +2027,6 @@ private function getTrustedValues(int $type, string $ip = null): array
         if ((self::$trustedHeaderSet & self::HEADER_FORWARDED) && (isset(self::FORWARDED_PARAMS[$type])) && $this->headers->has(self::TRUSTED_HEADERS[self::HEADER_FORWARDED])) {
             $forwarded = $this->headers->get(self::TRUSTED_HEADERS[self::HEADER_FORWARDED]);
             $parts = HeaderUtils::split($forwarded, ',;=');
-            $forwardedValues = [];
```

Comment:
Not relevant for this PR but PHPStorm told me that this is dead code ;) We can address this separately if desired but I don't want to create a PR for one line 🙈